great_expectations/rule_based_profiler/helpers/simple_semantic_type_filter.py

```diff
@@ -191,15 +186,13 @@ def _infer_semantic_domain_type_from_table_column_type(
         else:
             semantic_column_type = SemanticDomainTypes.UNKNOWN
 
-        inferred_semantic_column_type: InferredSemanticDomainType = (
-            InferredSemanticDomainType(
-                semantic_domain_type=semantic_column_type,
-                details={
-                    "algorithm_type": "deterministic",
-                    "mechanism": "lookup_table",
-                    "source": "great_expectations.profile.base.ProfilerTypeMapping",
```

Comment:
Where does this`source` change from 
`great_expectations.profile.base.ProfilerTypeMapping` to `"great_expectations.core.profiler_types_mapping.ProfilerTypeMapping"` come from?

Reply:
@Shinnnyshinshin It is because we moved the module, so I corrected its location in this metadata.  Thanks!